python_modules/dagster-graphql/dagster_graphql/implementation/fetch_partition_sets.py

```diff
@@ -229,6 +229,7 @@ def get_partition_set_partition_statuses(
 
     run_partition_data = graphene_info.context.instance.run_storage.get_run_partition_data(
         runs_filter=RunsFilter(
+            statuses=[status for status in DagsterRunStatus if status != DagsterRunStatus.CANCELED],
```

Comment:
maybe we should also include "canceling" runs?

Reply:
Ahh I was debating - I think that makes sense as well, will change this!